infrastructure/kube/templates/tbtc-maintainers/initcontainer/provision-tbtc-maintainers/tbtc-maintainers-config-template.toml

```diff
@@ -0,0 +1,27 @@
+# This is a TOML configuration file.
+
+# Connection details of a Electrum Server.
+[electrum]
+  ServerHost  = "tbtc-electrumx-server.tbtc.svc.cluster.local"
```

Comment:
Not sure if the `tbtc` here is related to the pod's namespace or not (i.e. if I should change it to `default`).